pkgs/development/libraries/jemalloc/default.nix

```diff
@@ -1,18 +1,24 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl,
+  # jemalloc is unable to correctly detect transparent hugepage support on
+  # ARM, and the default kernel ARMv6/7 kernel does not enable it, so we
+  # explicitly disable support
+  thpSupport ? stdenv.system != "armv7l-linux" &&
```

Comment:
Use: `stdenv.isArm`.

Reply:
I'd also add a link to https://github.com/jemalloc/jemalloc/issues/526 in the comments, so we know when we can strip this.